spark3-runtime/NOTICE

```diff
@@ -495,3 +495,128 @@ file:
 |   * HOMEPAGE:
 |     * http://www.opensource.apple.com/source/configd/configd-453.19/dnsinfo/dnsinfo.h
 
+--------------------------------------------------------------------------------
+
+This binary artifact includes Project Nessie with the following in its NOTICE
+file:
+
+| Dremio
+| Copyright 2015-2017 Dremio Corporation
+|
+| This product includes software developed at
+| The Apache Software Foundation (http://www.apache.org/).
```

Comment:
I don't think that we typically need the copyright part, or acknowledgement that there is ASF software. Since this is boilerplate and the relevant information is in LICENSE, we can probably omit it.